reloj-checador/Gemfile

```diff
@@ -32,6 +32,13 @@ gem 'bootsnap', '>= 1.4.4', require: false
 gem 'rspec-rails'
 gem 'devise'
 
+#generate xls
+gem 'axlsx', '= 2.0.1'
+gem 'axlsx_rails'
+
+gem 'will_paginate', '~> 3.1.0'
+gem 'will_paginate-bootstrap4'
+
```

Comment:
When adding a new dependency, worth committing in isolation and explain in the commit message why are those dependencies needed or added 😄 

That would help other teammates to understand why are we adding them and not only teammates but future contributors.

I know this is just a kata/excersice. But consider this for future contributions 😄

Reply:
Thank you so much, we will consider that in the future😊